vumi/transports/vumi_bridge/vumi_bridge.py

```diff
@@ -323,10 +231,21 @@ def handle_raw_inbound_event(self, request):
             msg = TransportEvent(_process_fields=True, **to_kwargs(data))
             yield self.handle_inbound_event(msg)
             request.finish()
+            if msg.payload["event_type"] == "ack":
+                self.update_status(
+                    status='ok', component='sent-by-vumi-go',
+                    type='vumi_go_sent', message='Sent by Vumi Go')
+            else:
```

Comment:
@hodgestar Didn't we decide we weren't going to handle delivery reports at this juncture? Although if this is all that's required I'm happy to add them.
What would a `nack` event_type mean and what should we do when we receive them?